Library/Homebrew/test/migrator_spec.rb

```diff
@@ -0,0 +1,269 @@
+require "migrator"
+require "test/support/fixtures/testball"
+require "tab"
+require "keg"
+
+class Migrator
+  describe self do
```

Comment:
I'd much rather you just referred to them as `Migrator::MigratorNoOldnameError` — every line it's used on is already pretty short, and I think this pattern is confusing.